nupic/frameworks/opf/clamodel.py

```diff
@@ -419,7 +423,7 @@ def run(self, inputRecord):
     if self.__logger.isEnabledFor(logging.DEBUG):
       self.__logger.debug("inputRecord: %r, results: %r" % (inputRecord,
                                                             results))
-
+    results.iterationTime = time.clock() - startTime
```

Comment:
actually, after a long time let to lay off, I came with a clean idea: 
How inaccurate do you think it would be if the time is measured:
* between each invocation of `metric.compute()` (or how is it called)
* instead of in `CLAModel.run()` at start-end of the method? 

This would radically simplify this metric/PR and make it stand-alone. Would that make sence? @scottpurdy @chetan51